File: backend/src/main/java/com/example/stamp_app/config/BeanConfig.java
package com.example.stamp_app.config;

import com.example.stamp_app.common.interceptor.AppInterceptor;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class BeanConfig {
    @Bean
    public AppInterceptor appInterceptor() throws Exception {
        return new AppInterceptor();
    }
}

File: backend/src/main/java/com/example/stamp_app/config/BeanConfig.kt
package com.example.stamp_app.config

import com.example.stamp_app.common.interceptor.AppInterceptor
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class BeanConfig {
	@Bean
	@Throws(Exception::class)
	fun appInterceptor(): AppInterceptor {
		return AppInterceptor()
	}
}
